helm/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "nuclei.fullname" . }}
+  labels:
+    {{- include "nuclei.labels" . | nindent 4 }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "nuclei.fullname" . }}
```

Comment:
can you please updae the name from `{{ include "nuclei.fullname" . }}` to `{{ include "nuclei.fullname" . }}-interactsh` ?
interactsh deployment name is wrong here